newrelic/provider.go

```diff
@@ -120,6 +120,7 @@ func Provider() terraform.ResourceProvider {
 			"newrelic_synthetics_monitor_script":    resourceNewRelicSyntheticsMonitorScript(),
 			"newrelic_synthetics_label":             resourceNewRelicSyntheticsLabel(),
 			"newrelic_synthetics_secure_credential": resourceNewRelicSyntheticsSecureCredential(),
+			"newrelic_app_label":                    resourceNewRelicAppLabel(),
```

Comment:
Please update to `newrelic_application_label` to be consistent with the other resources, and sort within the map instead of appending to the end.

Reply:
Renamed and reordered as suggested.